### PR TITLE
Add MATIC, INST and JRT as collateral 

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: Collateral Omnibus.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- UMIP <#> 
+- UMIP title: <Collateral Omnibus 4>
+- Author Chandler De Kock (chandler@umaproject.org)
+- Status: <Draft> 
+- Created: <15 July 2021>
+- Discourse Link: <Link>
+
+## Summary (2-5 sentences)
+
+This UMIP proposes adding **BADGER**, **OHM**, **IDLE**, **GNO**, **Qi**, **POOL**, **DOUGH V2**, **FEI**, **TRIBE** and **FOX** for use as collateral in UMA contracts.
```
Comment (Reinis-FRP) [inline Collateral Omnibus.md:12]: These collateral currencies were approved in [UMIP-116](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-116.md)

**Review by iluildephonse — APPROVED** [review]

**Review by therealhexi — COMMENTED** [review]
MATIC looks good to me

File: Omnibus5.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+**UMIP #**  - tbd
+
+-   **UMIP title:** Add MATIC and INST as collateral currency 
+-   **Author**  Chandler De Kock (chandler@umaproject.org)
+-   **Status: Draft**
+-   **Created:**  Date
+-   **Discourse Link:**  Insert link to discourse topic after it has been moved into draft UMIPs
```
Comment (smb2796) [inline Omnibus5.md:7]: You should add a discourse link or remove this

File: Omnibus5.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+**UMIP #**  - tbd
+
+-   **UMIP title:** Add MATIC and INST as collateral currency
```
Comment (smb2796) [inline Omnibus5.md:3]: Can you add the full token names as well here and in `Technical Specifications`?

File: Omnibus5.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+-   The WMATIC [Polygon token address](https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270): 0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270
+-   A final fee of **380 MATIC** needs to be added for Store in the contract.
+
+### InstaDapp token
+-   The INST [Ethereum token address](https://etherscan.io/address/0x6f40d4a6237c257fff2db00fa0510deeecd303eb): 0x6f40d4a6237c257fff2db00fa0510deeecd303eb
+-   The INST [Polygon token address](https://polygonscan.com/address/0xf50D05A1402d0adAfA880D36050736f9f6ee7dee): 0xf50D05A1402d0adAfA880D36050736f9f6ee7dee
+-   A final fee of **45 INST** needs to be added for Store in the contract.
+
+### Jarvis reward token
+-   The JRT [Ethereum token address](https://etherscan.io/address/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270): 0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270
```
Comment (Reinis-FRP) [inline Omnibus5.md:32]: This is not JRT address, but WMATIC on Polygon

**Review by Reinis-FRP — APPROVED** [review]
Looks good now

**Pull request merged**